content/about/charter.md

```diff
@@ -39,43 +41,49 @@ Responsibilities of each officer of the board are as follows:
 ### Secretary
 
 - Ensure that someone is assigned to and takes minutes during all
-    board EC meetings
-- Revise minutes for publication
+    board meetings
 - Report decisions and action items in formal minutes published
```

Comment:
Since it doesn't need to be the secretary who takes minutes (first point), how about: 
```suggestion
- Ensure that decisions and action items are reported in formal minutes published publicly
```